ls_settings.ino

```diff
@@ -363,7 +363,7 @@ void storeSettingsToPreset(byte p) {
 // The first time after new code is loaded into the Linnstrument, this sets the initial defaults of all settings.
 // On subsequent startups, these values are overwritten by loading the settings stored in flash.
 void initializeDeviceSettings() {
-  Device.version = MICROLINN_BASE_VERSION + 16;      // 16 is the latest non-MicroLinn version (created in 2023)
+  Device.version = 16 + MICROLINN_VERSION_OFFSET;  // 16 is the latest non-MicroLinn version (created in 2023)
```

Comment:
Looks like we both defined the MICROLINN version independently.  Doesn't matter which name we go with, but we should probably not have duplicate constants.